services/lighthouse/src/main.rs

```diff
@@ -120,12 +346,18 @@ async fn main() {
         .expect("failed to create database client");
 
     // create context
-    let ctx = LighthouseCtx { db: prisma.into() };
+    let prisma_locked = Arc::new(RwLock::new(prisma));
+    let ctx = LighthouseCtx {
+        db: prisma_locked.clone(),
+    };
 
     // create app router
     let app = Router::new()
         .route("/", get(report_status))
-        // .route("/endpoints", post(register_endpoint))
+        .route("/register", post(register_endpoint))
+        .route("/lookup", post(lookup_endpoint))
+        .route("/listconns", post(list_conns))
+        .route("/wipe", get(wipe_all)) // Testing purposes
```

Comment:
Might be nice to REST these?

Reply:
tbh all of them are creating new entries so I think POST requests are fine

Reply:
Okems